src/transport/MessageHeader.cpp

```diff
@@ -31,7 +31,6 @@
  * Header format (little endian):
  *
  *  16 bit: | VERSION: 4 bit | FLAGS: 4 bit | RESERVED: 8 bit |
- *  16 bit: | Secure message type                             |
  *  32 bit: | MESSAGE_ID                                      |
  *  32 bit: | Secure Session ID                               |
```

Comment:
Just because we don't have a spec that describes this I am going to ask here:
 - What about messages that are not encrypted and the session doesn't exist?
 - What about messages that are encrypted using Group Keys?
 - Can we use 16-bit identifier?

Just for the reference in OpenWeave we have an equivalent field "KeyID":
 - 16-bit value
 - KeyID = 0 means that the message is not encrypted
 - 4-bits of the value are used to encode the type of the key: session key, type of a group key...

Reply:
@balducci-apple what are your thoughts about this? We can probably address these in the spec document.